DSNS-262:   Updated the README with accurate instructions to build the package (#83)
GeoscienceAustralia/dea-ard-scene-select: README.rst

----------------------------
Modules are built off the module-prod branch. Create an annotated tag to tag a module build.

1. make sure the latest changes in the master branch are brought/sync-ed to the module-prod branch. To do this, create a PR from the master to the module-prod branch
2. once the PR is approved, merge it to the module-prod 

> 3. login or sudo as lpgs in a terminal. This is because production modules can must be built as the lpgs user

Reviewer: ```suggestion
3. login or sudo as lpgs in a terminal since production modules must be built as the lpgs user
```

4. head to the path, "/home/547/lpgs/sandbox/dea-ard-scene-select/module". Run "cd /home/547/lpgs/sandbox/dea-ard-scene-select/module/"

> 5. checkout the "module-prod" branch. Run "git checkout module-prod"

Reviewer: ```suggestion
5. checkout the relevant branch e.g. "git checkout master"
```
The go.sh script doesn't have to be from the module-prod branch

Author: you're right. I think a better way to word it is to say that the master branch needs to be checked out.
Checking out module-prod will not be as effective as the master branch will have the latest iteration of the go.sh script.

> 6. tag the new version. Here's what I ran for the most recent build:

Reviewer: ```suggestion
6. tag the new version. This does not have to be done as lpgs. For example:
```
This does not have to be done as lpgs - that's my understanding anyhow. But tag's haven't been working for me.

      git tag -a "ard-scene-select-py3-dea/20231010" -m "DSNS 262-baked pytest into scene select so that the new integration tests are supported"

7. build the new version of the package. Run "./go.sh --prod"

> 8. If there are no errors in the terminal, the package build should be have been successful and the

Reviewer: ```suggestion
8. If there are no errors in the terminal, the package build should have been successful and the
```

Author: haha sorted

> final line which resembles the example below will disclose where the newly built dea-ard-scene-select package as been built and written to.

Reviewer: ```suggestion
final line  is where the newly built dea-ard-scene-select package has been written to. For example,
```
